package.json

```diff
@@ -3,6 +3,13 @@
   "version": "1.0.0-beta.2",
   "description": "A normalized and configurable cache exchange for urql",
   "repository": "https://github.com/kitten/urql-exchange-graphcache",
+  "keywords": [
+    "normalised-cache",
```

Comment:
small nit: let's americani**z**e this and drop the `-` as well 😅 

> normalized cache

Reply:
This made me realise that our pkg.json has more wrong fields